modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManagerImpl.java

```diff
@@ -295,8 +295,9 @@ private void removeCacheData(int cacheId) {
             if (grp.sharedGroup()) {
                 assert cacheId != CU.UNDEFINED_CACHE_ID;
 
-                for (CacheDataStore store : cacheDataStores())
+                for (CacheDataStore store : cacheDataStores()) {
```

Comment:
Braces should not be used with one line statements

Reply:
Missed that. Will fix it later.